backend/app/routers/datasets.py

```diff
@@ -557,6 +567,7 @@ async def delete_folder(
     db: MongoClient = Depends(dependencies.get_db),
     fs: Minio = Depends(dependencies.get_fs),
     es: Elasticsearch = Depends(dependencies.get_elasticsearchclient),
+    allow: bool = Depends(DatasetAuthorization("owner")),
```

Comment:
i think it is OK to let editors do this also

Reply:
These are fixed. I don't use the allow yet but I am working on that, will have some questions at the meeting tomorrow.